css/style.css

```diff
@@ -57,6 +61,48 @@ a:hover {
   padding: 15px 20px;
 }
 
+.mobile-menu-background {
+  background: #e5e5e5;
+  height: 100vh;
+  z-index: 1;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+}
```

Comment:
- [ ] [optional] I am marking this change as optional because it was missed by the previous reviewer but I strongly suggest considering them. Please increase the gap between mobile menu options, It will make sure the project matches with the Template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/167481548-b881765a-dab2-47cc-8489-809757cd31f8.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/167481585-02a57385-b026-4fd3-b7cd-ba1b4daba66d.png)

Reply:
@thecodechaser I have reduced the space between the menu items because, when I am trying responsive mode in the browser, I feel the Figma design space is too much. This less place I think is actually looking good if you consider both side by side.
Anyways thanks for the suggestion, later I will try out once again with more space.